conf/server/server-postgres.conf

```diff
@@ -0,0 +1,54 @@
+server {
```

Comment:
I found this config helpful for running tests with local PostgreSQL, but I'm happy to remove it.